tck/src/test/resources/Token1.json

```diff
@@ -3,8 +3,8 @@
     "jti": "a-123",
     "sub": "24400320",
     "upn": "jdoe@example.com",
-    "preferred_username": "jdoe",
-    "aud": "s6BhdRkqt3",
+    "preferred_username": "jdoe",    
+    "aud": ["s6BhdRkqt3","aud1", "aud2", "aud3", "aud4"],
```

Comment:
`aud` can be both a string containing a single value and an array as in this PR. We should keep `Token1.json` resource unchanged and introduce `TokenAudiences.json` where `aud` is this array.

Reply:
done